assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -86,16 +81,14 @@ export default function SettingsForm() {
 				</div>
 			) }
 
-			{ ( isGA4Enabled && ( SETUP_FLOW_MODE_GA4_TRANSITIONAL === setupFlowMode || SETUP_FLOW_MODE_GA4 === setupFlowMode ) ) && (
-				<GA4PropertyNotice notice={ __( 'You need to connect the Google Analytics 4 property that’s associated with this Universal Analytics property.', 'google-site-kit' ) }>
-					<div className="googlesitekit-setup-module__inputs">
-						<GA4PropertySelect />
-					</div>
-				</GA4PropertyNotice>
+			{ ( isGA4Enabled && isGA4Connected ) && (
+				<div className="googlesitekit-setup-module__inputs">
```

Comment:
I think maybe we should add a class here to place the select below in the same position as the input for the new view name. It's a little odd that this one sits lower than that one.

Reply:
Updated.